test-cases/PR-provision-test.yaml

```diff
@@ -32,3 +32,6 @@ use_preinstalled_scylla: true
 post_behavior_db_nodes: "destroy"
 post_behavior_loader_nodes: "destroy"
 post_behavior_monitor_nodes: "destroy"
+
+append_scylla_yaml:
+  enable_tablets: false  # counters are not supported with tablets
```

Comment:
How about adding `no_tablets.yaml` into the pipeline instead?

Reply:
what difference does it make in this context?

Reply:
If I look at the pipeline (which is first point of contact with the test for me) I know immediately it has tablets turned off, this way it is somewhere down in the configs.

Also I do not see any counters mention in this config, so I do not think it is important to this config, it is important to the pipeline

Reply:
actually, this test config use counters in `stress_cmd` - so it is related specifically to this one

Reply:
Ah ok, sorry I totally missed that one. In that case, it is alright